<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Symbol</class>
 <widget class="QWidget" name="Symbol">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="not1">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>~</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="and1">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>∧</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="or1">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>∨</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="not2">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>`</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="and2">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>×</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="or2">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QPushButton" name="leftbracket">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>(</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QPushButton" name="rightbracket">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="sum">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>∑</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="prod">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>∏</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="key_M">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>M</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="key_m">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>m</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="key_D">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>D</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="equal">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QPushButton" name="comma">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>,</string>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QPushButton" name="pushButton_24">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="condition">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>→</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="doublecondition">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>↔</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="nand">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>↑</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="nor">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>↓</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QPushButton" name="key_1">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QPushButton" name="key_0">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="7">
      <widget class="QPushButton" name="key_dash">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="key_A">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>A</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="key_B">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>B</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="key_C">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>C</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="key_D_2">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>D</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="key_X">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QPushButton" name="key_Y">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QPushButton" name="key_Z">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Z</string>
       </property>
      </widget>
     </item>
     <item row="3" column="7">
      <widget class="QPushButton" name="key_W">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>W</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="key_xor">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>110</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>⊕</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
